### PR TITLE
PSGEN-280-Module_generator_full_port_to_v6

### PR DESCRIPTION
**Pull request by mantas-kemesius**

### CODE REVIEW

File: Application/Controller/Admin/Admin_oxpsModuleGenerator.php [hunk 1]
```diff
@@ -366,7 +375,7 @@ protected function _getFormVersionFieldValue($sRequestVersion)
         $sModuleVersion = trim((string) $sRequestVersion);
 
         if (empty($sModuleVersion)) {
-            $sModuleVersion = '1.0.0';
+            $sModuleVersion = '2.0.0';
```
Comment (druteika) [inline Application/Controller/Admin/Admin_oxpsModuleGenerator.php]: Default module version should remain '1.0.0'

File: Core/Module.php [hunk 2]
```diff
@@ -24,14 +24,18 @@
  * @copyright (C) OXID eSales AG 2003-2017
  */
 
-use \OxidEsales\Eshop\Core\Module\Module;
+namespace Oxps\ModuleGenerator\Core;
+
+use OxidEsales\Eshop\Application\Model\Content;
+use OxidEsales\Eshop\Core\Model\MultiLanguageModel;
+use \OxidEsales\Eshop\Core\Module\Module as BaseModel;
```
Comment (druteika) [inline Core/Module.php]: Perhaps better naming would be "BaseModule"

**Review by druteika — CHANGES_REQUESTED** [review]

File: Application/Controller/Admin/Admin_oxpsModuleGenerator.php (hunk 1 above)
Comment (mantas-kemesius) [reply Application/Controller/Admin/Admin_oxpsModuleGenerator.php]: Change module version to '1.0.0'

File: Core/Module.php (hunk 2 above)
Comment (mantas-kemesius) [reply Core/Module.php]: Change naming from "BaseModel" => "BaseModule"

**Review by druteika — APPROVED** [review]

**Pull request merged**